// react-modal-image/es/Header.js
import React from "react";

import { header, iconMenu, icon, caption } from "./styles";

import { ZoomInIcon, ZoomOutIcon, DownloadIcon, CloseIcon } from "./icons";

var Header = function Header(_ref) {
  var image = _ref.image,
      alt = _ref.alt,
      zoomed = _ref.zoomed,
      toggleZoom = _ref.toggleZoom,
      onClose = _ref.onClose;
  return React.createElement(
    "div",
    { style: header },
    React.createElement(
      "span",
      { style: iconMenu },
      React.createElement(
        "a",
        {
          className: "react-modal-image-download",
          href: image,
          style: icon,
          download: true
        },
        React.createElement(DownloadIcon, null)
      ),
      React.createElement(
        "a",
        {
          className: "react-modal-image-zoom",
          href: "",
          style: icon,
          onClick: toggleZoom
        },
        zoomed ? React.createElement(ZoomOutIcon, null) : React.createElement(ZoomInIcon, null)
      ),
      React.createElement(
        "a",
        { className: "react-modal-image-close", style: icon, onClick: onClose },
        React.createElement(CloseIcon, null)
      )
    ),
    alt && React.createElement(
      "span",
      { style: caption },
      alt
    )
  );
};

export default Header;
