// react-modal-image/lib/Header.js
"use strict";

exports.__esModule = true;

var _react = require("react");

var _react2 = _interopRequireDefault(_react);

var _styles = require("./styles");

var _icons = require("./icons");

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

var Header = function Header(_ref) {
  var image = _ref.image,
      alt = _ref.alt,
      zoomed = _ref.zoomed,
      toggleZoom = _ref.toggleZoom,
      onClose = _ref.onClose;
  return _react2.default.createElement(
    "div",
    { style: _styles.header },
    _react2.default.createElement(
      "span",
      { style: _styles.iconMenu },
      _react2.default.createElement(
        "a",
        {
          className: "react-modal-image-download",
          href: image,
          style: _styles.icon,
          download: true
        },
        _react2.default.createElement(_icons.DownloadIcon, null)
      ),
      _react2.default.createElement(
        "a",
        {
          className: "react-modal-image-zoom",
          href: "",
          style: _styles.icon,
          onClick: toggleZoom
        },
        zoomed ? _react2.default.createElement(_icons.ZoomOutIcon, null) : _react2.default.createElement(_icons.ZoomInIcon, null)
      ),
      _react2.default.createElement(
        "a",
        { className: "react-modal-image-close", style: _styles.icon, onClick: onClose },
        _react2.default.createElement(_icons.CloseIcon, null)
      )
    ),
    alt && _react2.default.createElement(
      "span",
      { style: _styles.caption },
      alt
    )
  );
};

exports.default = Header;
module.exports = exports["default"];
