// react-modal-image/lib/styles.js
"use strict";

exports.__esModule = true;
var smallImage = exports.smallImage = {
  cursor: "pointer",
  maxWidth: "100%",
  maxHeight: "100%"
};

var modal = exports.modal = {
  position: "fixed",
  zIndex: 1000,
  left: 0,
  top: 0,
  width: "100%",
  height: "100%",
  overflow: "auto",
  backgroundColor: "rgba(0, 0, 0, 0.8)",
  touchAction: "none"
};

var modalContent = exports.modalContent = {
  textAlign: "center",
  position: "relative",
  height: "100%",
  width: "100%",
  overflow: "hidden"
};

var spinner = exports.spinner = {
  position: "absolute",
  top: "50%",
  left: "50%",
  transform: "translate(-50%, -50%)"
};

var mediumImage = exports.mediumImage = {
  position: "absolute",
  top: "50%",
  left: "50%",
  transform: "translate3d(-50%, -50%, 0)",
  maxWidth: "98%",
  maxHeight: "98%"
};

var largeImage = exports.largeImage = function largeImage(x, y) {
  return {
    position: "absolute",
    top: "50%",
    left: "50%",
    transform: "translate3d(-50%, -50%, 0) translate3d(" + x + "px, " + y + "px, 0)",
    cursor: "move"
  };
};

var icon = exports.icon = {
  display: "inline-block",
  fontSize: "40px",
  cursor: "pointer",
  lineHeight: "40px",
  boxSizing: "border-box",
  border: "none",
  padding: "0px 5px 0px 5px",
  marginLeft: "10px",
  color: "white",
  backgroundColor: "rgba(0, 0, 0, 0)"
};

var iconMenu = exports.iconMenu = {
  display: "inline-block",
  float: "right"
};

var caption = exports.caption = {
  display: "inline-block",
  color: "white",
  fontSize: "120%",
  padding: "10px",
  margin: "0"
};

var header = exports.header = {
  position: "absolute",
  top: 0,
  width: "100%",
  backgroundColor: "rgba(0, 0, 0, 0.7)"
};
